clickhouse_connect/driver/httputil.py

```diff
@@ -0,0 +1,84 @@
+import atexit
+import http
+import sys
+import socket
+from typing import Optional
+
+from requests import Response
+from requests.adapters import HTTPAdapter
+from urllib3 import poolmanager
+
+
+# Increase this number just to be safe when ClickHouse is returning progress headers
+http._MAXHEADERS = 10000  # pylint: disable=protected-access
+
+KEEP_INTERVAL = 30
+KEEP_COUNT = 3
+KEEP_IDLE = 30
```

Comment:
I guess these are default values. If yes, would it make sense to have it with `DEFAULT_` prefix?